src/Snackbar/Snackbar.js

```diff
@@ -292,6 +297,18 @@ Snackbar.propTypes = {
    * @ignore
    */
   className: PropTypes.string,
+  /**
+   * If `false`, the automatic calling of the `onClose` function after `autoHideDuration`
+   * is paused as long as the window is not focused.
+   * When the window regains focus, the configured `autoHideDuration` is counted down again.
+   *
+   * If `true`, the `autoHideDuration` will be counted down even if the window is not focused.
+   * The `onClose` function is called after the `autoHideDuration` even if the window is not
+   * focused.
+   *
+   * If `autoHideDuration` property isn't specified, it does nothing.
+   */
```

Comment:
"If `true`, the `autoHideDuration` timer will expire even if the window is not focused." is sufficient. Everything else can be inferred from that.

Reply:
Done. :+1: